site/data/examples.yml

```diff
@@ -18,9 +18,11 @@
 #     - name: Modals
 #       description: "Transform modals to serve any purpose, from feature tours to dialogs."
 
-# - category: Custom Components
-#   description: "Brand new components and templates to help folks quickly get started with Boosted and demonstrate best practices for adding onto the framework."
-#   examples:
+- category: Custom Components
```

Comment:
We probably should remove the "Components" category and move its content in this one. Bootstrap doesn't have "Components".